regnety/dataset/imagenet.py

```diff
@@ -214,6 +259,7 @@ def center_crop_224(self, image: tf.Tensor, target: tf.Tensor) -> tuple:
         """
         aug_images = tf.image.resize(image, (320, 320))
         aug_images = tf.image.central_crop(aug_images, 224./320.)
+        aug_images = tf.image.resize(image, (320, 320))
```

Comment:
Prefer having these values in configurations. That way it's comprehensive to the users.

Reply:
Actually, there is no other function that requires information a config file. Is it okay of I include `crop_size` argument in the constructor, and set it to a default value? 
I prefer if the input pipeline takes no config file, so that it can be used independently. But please tell me if it's better otherwise, I'll change it.

Reply:
I think having a separate configuration file for the input pipeline would make it more customizable.

Reply:
Sure, no problem. I'll make a dataclass for that.